website/templates/components/dashboard_templates.mako

```diff
@@ -140,7 +140,7 @@
             <br />
 
             <label>Description (Optional)</label>
-            <textarea data-bind="value: description"class="form-control" name="description"
+            <textarea data-bind="value: description"class="form-control newDiscription" name="description"
```

Comment:
Perhaps you could rename this as reusable utility class, instead of making it specific to this description input.  `resize-vertical`?